shrine/lib/backdrop.dart

```diff
@@ -74,7 +76,10 @@ class _BackdropTitle extends AnimatedWidget {
 
   @override
   Widget build(BuildContext context) {
-    final Animation<double> animation = this.listenable;
+    final Animation<double> animation = CurvedAnimation(
+      parent: this.listenable,
+      curve: Interval(0.0, 0.78),
```

Comment:
Is there a reason 0.78 doesn't get a constant like the ones in lines 20-23?

Reply:
Usually avoid constants in the educational code but they helped make the emphasized easing function easier to read.